assets/js/atomic/blocks/product-elements/product-meta/index.tsx

```diff
@@ -18,7 +18,14 @@ registerBlockSingleProductTemplate( {
 	blockSettings: {
 		edit,
 		save,
-		icon,
+		icon: {
+			src: () => (
+				<img
+					src={ `${ WC_BLOCKS_IMAGE_URL }/blocks/product-meta/product-meta-icon.svg` }
+					alt="Product Meta Icon"
```

Comment:
According to [guidelines for using decorative images](https://www.w3.org/WAI/tutorials/images/decorative/) 

> Decorative images don’t add information to the content of a page. For example, the information provided by the image might already be given using adjacent text, or the image might be included to make the website more visually attractive.
> In these cases, a null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers.

we should skip the `alt` so it's ignored by readers as the icon doesn't bring new information and there's an actual Block title with a description next to it. What do you think?

Reply:
TIL!